struct.go

```diff
@@ -74,22 +74,21 @@ type RegisterInstanceJson struct {
 	Instance *Instance `json:"instance"`
 }
 
-// Instance [de]serializeable [to|from] Eureka XML.
+// Instance [de]serializeable [to|from] Eureka [XML|JSON].
 type Instance struct {
-	XMLName          struct{} `xml:"instance" json:"-"`
-	HostName         string   `xml:"hostName" json:"hostName"`
-	App              string   `xml:"app" json:"app"`
-	IPAddr           string   `xml:"ipAddr" json:"ipAddr"`
-	VipAddress       string   `xml:"vipAddress" json:"vipAddress"`
-	SecureVipAddress string   `xml:"secureVipAddress" json:"secureVipAddress"`
+	HostName         string `xml:"hostName" json:"hostName"`
+	App              string `xml:"app" json:"app"`
+	IPAddr           string `xml:"ipAddr" json:"ipAddr"`
+	VipAddress       string `xml:"vipAddress" json:"vipAddress"`
+	SecureVipAddress string `xml:"secureVipAddress" json:"secureVipAddress"`
 
 	Status           StatusType `xml:"status" json:"status"`
 	Overriddenstatus StatusType `xml:"overriddenstatus" json:"overriddenstatus"`
 
-	Port        int  `xml:"port" json:"-"`
-	PortJ       Port `json:"port" xml:"-"`
-	SecurePort  int  `xml:"securePort" json:"-"`
-	SecurePortJ Port `json:"securePort" xml:"-"`
+	Port              int  `xml:"-" json:"-"`
+	PortEnabled       bool `xml:"-" json:"-"`
```

Comment:
@seh Don't worry about [cfkit](http://www.github.com/stv3v/cfkit) and [pcf-micro-examples](https://github.com/st3v/pcf-micro-examples). Those were pet projects I played around with quite a while ago. They aren't used anywhere.